playwright/tests/authorSearch.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Given, Then, When } from "@cucumber/cucumber";
+import { expect } from "@playwright/test";
+import { CustomWorld } from "../support/setup";
+
+Given("I want to do an author search", async function (this: CustomWorld) {
+    return await this.page.goto("https://digital-research-books-beta.nypl.org/");
+});
+When("I change the dropdown to author", async function (this: CustomWorld) {
+    const dropdown = await this.page.$("#searchbar-select-search-bar");
```

Comment:
Change to use `aria-label` instead of id

Reply:
done.